### ISSUE TITLE
Seems `load_obj` can not load `vertices` with `vertex-color`

### ISSUE THREAD

**Issue by FavorMylikes** [facebookresearch/pytorch3d#1265]
I have an obj file that includes `vertex-color`.

It looks like

```obj
vn 0.228076 -5.246246 2.607272
v -0.053804 0.709615 -0.144573 0.235294 0.992157 0.125490
vn -0.542669 -4.832255 3.054440
v -0.056966 0.711699 -0.141544 0.235294 0.992157 0.125490
...
```

## How did this file come from

- Download the `ply` [file](https://storage.googleapis.com/tensorflow-graphics/tensorboard/test_data/ShortDance07_a175_00001.ply)
  - And I found it from the [official tutorial](https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/tensorboard/plugins/mesh/Mesh_Plugin_Tensorboard.ipynb) `Mesh plugin`
- Open it in `MeshLab`
- Export it to the `obj` file

## I checked the key method `_parse_obj`, as you can see below

https://github.com/facebookresearch/pytorch3d/blob/7978ffd1e4819d24803b01a1147a2c33ad97c142/pytorch3d/io/obj_io.py#L459-L464

And it seems can not load vertex-color, cause the code use `token[1:4]` only

**Comment by bottler** [comment]
You are right that we do not support vertex color in obj files. Let us know if you would like to contribute such support.

Just to respond to the title: the vertices are loaded but their colors are not.

**Comment by FavorMylikes** [comment]
Use `trimesh` as a temporary alternative.

```python
import trimesh
trimesh.load(obj_file)
colors = torch.Tensor(tri_mesh.visual.vertex_colors)  # [N, C], Channel = 4 for default.
```

Btw, `pytorch3d` can not load files by specific encoding.
